CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 # Change Log
+# [1.0.0 (April 15th, 2021)](https://github.com/eclipse/eclipse.jdt.ls/milestone/89?closed=1)
+ * feature - Support refactoring when renaming or moving files. See [#1445](https://github.com/eclipse/eclipse.jdt.ls/pull/1445).
+ * feature - Support Type Hierarchy. See [#1656](https://github.com/eclipse/eclipse.jdt.ls/pull/1656).
+ * feature - Support to specify source paths for unmanaged folder. See [#1658](https://github.com/eclipse/eclipse.jdt.ls/pull/1658).
+ * feature - Allow to customize the insertion location for the code generated by source actions. See [#1713](https://github.com/eclipse/eclipse.jdt.ls/pull/1713).
+ * bug - External tool file modifications not registered. See [#1650](https://github.com/eclipse/eclipse.jdt.ls/issues/1650).
+ * bug - Enhance the condition of inline constant. See [#1672](https://github.com/eclipse/eclipse.jdt.ls/pull/1672).
+ * bug - Update formatter profile options before applying. See [#1675](https://github.com/eclipse/eclipse.jdt.ls/pull/1675).
+ * bug - v0.70 broke workspace/DidChangeConfiguration. See [#1685](https://github.com/eclipse/eclipse.jdt.ls/issues/1685).
+ * bug - Correct minor typo in exception message. See [#1687](https://github.com/eclipse/eclipse.jdt.ls/pull/1687).
+ * bug - Fall back to default value when import threashold is non-positive. See [#1715](https://github.com/eclipse/eclipse.jdt.ls/pull/1715).
+ * build - Update Target Platform to 2021-03 Release. See [#1691](https://github.com/eclipse/eclipse.jdt.ls/pull/1691).
+ * build - Revert Java 16 support for the 1.0 release. See [#1708](https://github.com/eclipse/eclipse.jdt.ls/pull/1708).
+ * build - Update lsp4j to 0.11.0. See [#1700](https://github.com/eclipse/eclipse.jdt.ls/pull/1700).
+ * debt - Build Fails Due to Tests. See [#1646](https://github.com/eclipse/eclipse.jdt.ls/issues/1646).
+ * debt - Stop generating .gz artifacts during builds. See [#1707](https://github.com/eclipse/eclipse.jdt.ls/issues/1707).
+ * debt - Java LS Tests fail randomly. See [#1684](https://github.com/eclipse/eclipse.jdt.ls/issues/1684).
+ * other - Support String formatting via delegate command. See [#1702](https://github.com/eclipse/eclipse.jdt.ls/pull/1702).
+ * other - Add more options to query project settings. See [#1682](https://github.com/eclipse/eclipse.jdt.ls/pull/1682).
```

Comment:
those are enhancements

Reply:
Yes. seems I called "features" enhancements. I'll fix this.

Reply:
Moved the 2 entries to enhancements.